pilot/pkg/xds/v3/model.go

```diff
@@ -62,7 +62,7 @@ func GetShortType(typeURL string) string {
 		return "PCDS"
 	case ExtensionConfigurationType:
 		return "ECDS"
-	case WorkloadType:
+	case AddressType:
 		return "WDS"
```

Comment:
WDS is kind of weird name now that its "address". ADS is already taken though...